torch/_inductor/compile_fx.py

```diff
@@ -615,6 +654,9 @@ def _compile_fx_inner(
             **graph_kwargs,
         )
 
+    if precompile := _get_precompile(graph_kwargs.get("name"), example_inputs):
+        return precompile
```

Comment:
This seems like an odd place to check the cache.  By this point we have already run dynamo and AOT Autograd, which incur a lot of compile time -- but then we just throw out the graph we worked so hard to generate.  If we moved this check up to the object returned by `torch.compile` then we could get the compile time down close to zero.